app/controllers/user_auth_controller.rb

```diff
@@ -32,4 +32,18 @@ def user_not_authorized(exception)
   def authenticated_user_root_path
     current_user.only_invited? ? root_path : users_rdvs_path
   end
+
+  def verify_user_identity
```

Comment:
```suggestion
  def verify_user_name_initials?
```

L’idée de différencier Devise de ce truc spécifique